expression/builtin_miscellaneous_vec_test.go

```diff
@@ -81,3 +82,19 @@ func BenchmarkVectorizedBuiltinMiscellaneousCasesEvalOneVec(b *testing.B) {
 func BenchmarkVectorizedBuiltinMiscellaneousCasesFunc(b *testing.B) {
 	benchmarkVectorizedBuiltinFunc(b, vecBuiltinMiscellaneousCases)
 }
+
```

Comment:
And please help us to remove these four methods above here(line 70~84);

Reply:
OK, I have removed.